instance/instance_external_test.go

```diff
@@ -401,7 +401,7 @@ func TestUpdateCompletedInstanceToCompletedReturnsForbidden(t *testing.T) {
 					ID: "4567",
 				},
 			},
-			State: models.CompletedState,
+			State: models.PublishedState,
```

Comment:
If we're changing this state we should probably change the test name to be consistent. Do we not feel there's value in tests for both cases?

Reply:
So firstly this is a duplicate test from the one above. Secondly if the instance is state is completed, resetting the state to completed will not cause an error. So I will remove test.